FlowCrypt/Controllers/Setup/SetupManuallyImportKeyViewController.swift

```diff
@@ -171,7 +171,10 @@ extension SetupManuallyImportKeyViewController {
         if privateKey.isEmpty {
             userInfoMessage = "import_no_backups_clipboard".localized + user
         } else {
-            userInfoMessage = "Found \(privateKey.count) key\(privateKey.count > 1 ? "s" : "")"
+            let key = privateKey.count > 1
```

Comment:
For such cases it's better to use `stringsdict` with pluralisation support - it'll automatically use appropriate localization for provided parameter (`privateKey.count`) - https://github.com/FlowCrypt/flowcrypt-ios/blob/master/FlowCrypt/Resources/Localizable.stringsdict

Reply:
Why I do not prefer this solution because `NSLocalizedString` accepts array of CVarArg as arguments and in case we pass string instead of int we will get a crash in runtime

As you see to prevent that I have changed implementation for localizeWithArguments to accept only stings in code and all parameters in `localization.strings` accepts only `%@` as String parameter
```
@inline(__always) func localizeWithArguments(_ arguments: String...)
``` 

With `stringsdict` solution we need to explicitly pass Int as a variable to be able to distinguish is it variable plural. 
Do we really want to have this possibility?

Reply:
@sosnovsky 
What do you think if I leave only Strings as parameters here in `localizeWithArguments` implementation? And use only strings in localizable files 

And will add a separate `func localizePluralWithArguments` to accept `Int` and use stringsdict?

Reply:
> @sosnovsky What do you think if I leave only Strings as parameters here in `localizeWithArguments` implementation? And we will use only string as a parameter in localizable files
> 
> And will add a separate `func localizePluralWithArguments` to accept `Int` and use stringsdict?

Sounds good for me, let's do it this way 👍